resources/metadata/abapEnvironmentRunATCCheck.yaml

```diff
@@ -0,0 +1,101 @@
+metadata:
+  name: abapEnvironmentRunATCCheck
+  description: Runs an ATC Check
+  longDescription: |
+    Run ATC Check
+spec:
+  inputs:
+    secrets:
+      - name: credentialsId
```

Comment:
Use tool specific name

Reply:
This can be used both for Cloud Foundry Credentials for Reading a Cloud Foundry ABAP Instance Service Key which contains the credentials and the endpoint URL for the ABAP system. 
Alternatively you can directly provide the ABAP endpoint in the host configuration with the respective credentials which can also be used in this flag.

Hence, this all depends on the type of way you chose that's why this flag/parameter should be a more general name IMO. `abapCredentialsId` might mislead users chosing the Cloud Foundry way.

Reply:
It would be good to name them accordingly (so you would have `cfCredentialsId` and `abapCredentialsId`) as other steps might also use CF or ABAP credentials in the pipeline.

Reply:
Is this possible now? When abapEnvironmentPullGitRepos was developed, it was not possible. Hence the generic "credentialsId". I'm not sure if we could change abapEnvironmentGitRepo in a compatible manner...

Reply:
As just discussed on the phone, lets go this way as other ABAP steps have the same approach.